### DISCUSSION TITLE
Teensy 4.0 HWSPI on GC09A01

### DISCUSSION THREAD

**Discussion by chester2400** [moononournation/Arduino_GFX#132] [Q&A] [answered]
Im having a problem using HWSPI on teensy 4.0.
The screen will not initialize on HWSPI. But if I run it on SWSPI and the reload the program with HWSPI without powering, down the screen works faster. On SWSPI I cant get more then 4-5 FPS without slowing other functions. And also standard servo library  interfers with the screen.

**Comment by moononournation** [comment, marked as answer]
what voltage you connected to the display?
And you may try lower the SPI frequency by: gfx->begin(8000000);

**Reply by chester2400** [reply]
The display runs on 3.3v. I will try the lower frequency. Thanx.
edit.
The lower frequency worked. I started to loose it around 35 .MHz
